packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -1128,6 +1139,12 @@ class SemanticsProperties extends DiagnosticableTree {
   ///  * [hint] for a plain string version of this property.
   final AttributedString? attributedHint;
 
+  /// Provides a textual description of the widget's tooltip.
+  ///
+  /// If a [tooltip] is provided, there must either by an ambient
+  /// [Directionality] or an explicit [textDirection] should be provided.
```

Comment:
Somewhere we should document that on iOS this is just read out as part of the label (or value?).